src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationServiceTest.java

```diff
@@ -99,6 +99,7 @@ void testCreateSendNotification() {
         assertEquals("John Doe", sendNotificationType.getSendNotificationFullName2());
         assertEquals("details", sendNotificationType.getSendNotificationDetails());
         assertEquals("Judge", sendNotificationType.getSendNotificationRequestMadeBy());
+        assertEquals("notStartedYet", sendNotificationType.getNotificationState());
```

Comment:
You've added an if statement with 2 branches, but only 1 test for this. Is there a branch not covered?